src/kpi_app/app.py

```diff
@@ -439,8 +433,8 @@ def create_card(title, content1, content2):
     return card
 
 
-def create_card_Autorinnen(edits=10, interest="all", subject="all"):
-    var_1 = Anzahl_Autorinnen(
+def create_card_authors(edits=10, interest="all", subject="all"):
```

Comment:
I understand that card with name Authorinnen is meant, but anyway we should keep it in English, the dev colleagues are going to understand when they read the body of the function.